src/components/RequestsFilters/PickupServicePointFilter/PickupServicePointFilter.test.js

```diff
@@ -0,0 +1,46 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import PickupServicePointFilter from './PickupServicePointFilter';
+
+jest.mock('@folio/stripes/components', () => ({
+  Accordion: jest.fn(({ name, children, onClearFilter }) => (
+    <div>
+      <div><button type="button" onClick={() => onClearFilter()} data-testid={`clear-${name}`}>Clear</button></div>
+      <div data-testid={`accordion-${name}`}>{children}</div>
+    </div>
+  )),
+  FilterAccordionHeader: jest.fn(() => <div>Filter Accordion Header</div>),
+}));
+
+jest.mock('@folio/stripes/smart-components', () => ({
+  MultiSelectionFilter: jest.fn(() => <div>Multi Selection Filter</div>)
+}));
```

Comment:
Could you add mocks to appropriate files here if they does not exist yet - `test/jest/__mock__` ?